### ISSUE TITLE
On Clicking on Send Getting the Error: Cannot read property zone on Null

### ISSUE THREAD

**Issue by bawa93** [MatheusrdSantos/vue-quick-chat#48]
I am just using the example provided on the readme page in my nuxtjs project
```
TypeError: Cannot read property 'zone' of null
    at Function.fromObject (vue-quick-chat.common.js:7236)
    at vue-quick-chat.common.js:17232
    at Array.map (<anonymous>)
    at Store.setMessages (vue-quick-chat.common.js:17230)
    at wrappedMutationHandler (vue-quick-chat.common.js:10860)
    at commitIterator (vue-quick-chat.common.js:10553)
    at Array.forEach (<anonymous>)
    at vue-quick-chat.common.js:10552
    at Store._withCommit (vue-quick-chat.common.js:10662)
    at Store.commit (vue-quick-chat.common.js:10551)
```

**Comment by vonec** [comment]
this happens when the message.timestamp is  null, this happens only when you attach an image. im debugging this behavior @MatheusrdSantos  can we touch base on this im about to send a PR with few changes

**Comment by MatheusrdSantos** [comment]
@vonec, yes. Today I'll take some time to fix some issues.

**Comment by MatheusrdSantos** [comment]
Fixed on version 1.2.7.
Thank you, @vonec!

**Comment by vonec** [comment]
i will be sending another PR in a while which would enable file attachments please hang around
